src/rgw/rgw_cache.cc

```diff
@@ -148,7 +146,7 @@ bool ObjectCache::chain_cache_entry(std::initializer_list<rgw_cache_entry_info*>
 
 void ObjectCache::put(const string& name, ObjectCacheInfo& info, rgw_cache_entry_info *cache_info)
 {
-  std::unique_lock l{lock};
```

Comment:
this is all making the code less modern c++ and adding legacy ceph noise

Reply:
apologies for the regresion, RWLock::* replaced later on with std::unique_lock